codenames-web-services/src/main/java/com/codenames/controllers/GameRoomsWebSocketController.java

```diff
@@ -81,41 +73,42 @@ public void connectToRoom(Request<Integer> request){
         roomService.addUserToRoom(room, player);
         this.authorizedUsers.setNewRoomID(request.getSessionID(), request.getRequestBody());
 
-        gameService.sendNewRoomInfoToPlayer(request, room, player);
+        gameService.sendRoomInfoToAllRoomPlayer(room);
     }
 
 
-    @SocketMapping(reqPath = "/room/create")
-    @SocketMappingFilters(filters = {
-            UserAuthorizedFilter.class
-    })
-    public void createNewRoom(Request<String> request){
-        // TODO: 18.04.2023  Add a check to see if the user has a room
-
-
-        int newRoomID = gameService.createNewRoom(codeNamesGame, playerService.getPlayerByRequest(request));
-        playerService.setPlayerRoomID(request, newRoomID);
-
-        Room room = gameService.getRoomByID(codeNamesGame, newRoomID);
-        Player player = playerService.getPlayerByRequest(request);
-
-        LOGGER.info("Created new room, id - " + newRoomID);
-
-        gameService.sendNewRoomInfoToPlayer(request, room, player);
-    }
+//    @SocketMapping(reqPath = "/room/create")
```

Comment:
Leave TODO or remove

Reply:
This code is not needed, so I removed it